common.sh

```diff
@@ -14,23 +17,26 @@ else
   HOMEBREW_PREFIX=/opt/homebrew
 fi
 
-cbr() {
+f5m_configure() {
   # Install plugins to /usr/local for both arm and x86 to ease fcitx5 search
-  PKG_CONFIG_PATH=$HOMEBREW_PREFIX/lib/pkgconfig cmake -B build -G Ninja \
+  PKG_CONFIG_PATH="$HOMEBREW_PREFIX/lib/pkgconfig:$PKG_CONFIG_PATH" cmake -B build -G Ninja \
     -DCMAKE_BUILD_TYPE=Release \
     -DCMAKE_FIND_ROOT_PATH="/Library/Input Methods/Fcitx5.app/Contents;$HOMEBREW_PREFIX" \
-    -DCMAKE_OSX_ARCHITECTURES=$ARCH "$@"
+    -DCMAKE_INSTALL_RPATH="/Library/Input Methods/Fcitx5-x86_64.app/Contents/lib" \
+    -DCMAKE_OSX_ARCHITECTURES=$ARCH \
+    -DCMAKE_INSTALL_PREFIX="$ADDON_INSTALL_PREFIX" \
```

Comment:
CMAKE_INSTALL_PREFIX and DESTDIR can't replace each other so this needs revert.

Reply:
CMake cannot handle `DESTDIR` correctly when finding libraries.

Reply:
Reverted in 9e81d01193592901ad57dd701ee80e47504eb525